packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx

```diff
@@ -39,32 +39,38 @@ const PatientChart: React.FC = () => {
 
   return (
     <main className={`omrs-main-content ${styles.chartContainer}`}>
-      <>
-        <div
-          className={`${styles.innerChartContainer} ${
-            windowSize.size === 'normal' && active ? styles.closeWorkspace : styles.activeWorkspace
-          }`}
-        >
-          <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
-          {isLoadingPatient ? (
-            <Loader />
-          ) : (
-            <>
-              <aside>
-                <ExtensionSlot extensionSlotName="patient-header-slot" state={state} />
-                <ExtensionSlot extensionSlotName="patient-info-slot" state={state} />
-              </aside>
-              <div className={styles.grid}>
-                <div className={styles.chartReview}>
-                  <ChartReview {...state} view={view} />
-                  <WorkspaceNotification />
+      {isLoadingPatient ? (
+        <Loader />
+      ) : (
+        <>
+          <div
+            className={`${styles.innerChartContainer} ${
+              windowSize.size === 'normal' && active ? styles.closeWorkspace : styles.activeWorkspace
+            }`}
+          >
+            <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
+            {isLoadingPatient ? (
+              <Loader />
+            ) : (
+              <>
+                <aside>
+                  <ExtensionSlot extensionSlotName="patient-header-slot" state={state} />
+                  <ExtensionSlot extensionSlotName="what-is-new-bar-slot" state={state} />
```

Comment:
How about we rename to: `patient-highlights-bar-slot` which will go in-line with the docs [here](https://zeroheight.com/23a080e38/p/082745-highlights-bar)